os/kernel/Kconfig

```diff
@@ -1080,6 +1080,26 @@ config SCHED_LPWORKSTACKSIZE
 endif # SCHED_LPWORK
 endmenu # Work Queue Support
 
+menu "fault Manager"
+config FAULT_MGR
+	bool "Enable Fault Manager"
+	default n
+	depends on APP_BINARY_SEPARATION
+	---help---
+		This interface is used to manage any fault occurs in a system.
+		It will either restart app or reboot the system according to
+		the configuration.
+
+if FAULT_MGR
+config DEBUG_FAULT_MGR
+	bool "Enable Debug logs of Fault Manager"
+	default n
+	depends on FAULT_MGR && DEBUG
```

Comment:
I think no need to add dependency again on FAULT_MGR.

Reply:
okay, will modify it.